articles/active-directory/develop/sample-v2-code.md

```diff
@@ -136,6 +136,15 @@ The following sample shows a public client application running on a device witho
 > | Java | [Sign in users and invoke protected API](https://github.com/Azure-Samples/ms-identity-java-devicecodeflow) | MSAL Java | Device code |
 > | Python | [Call Microsoft Graph](https://github.com/Azure-Samples/ms-identity-python-devicecodeflow) | MSAL Python | Device code |
 
+## Microsoft Teams applications
+
+The following samples illustrate Microsoft Teams applications that sign in users. Some samples also demonstrate how to call Microsoft Graph, or your own web API with the user's identity using the Microsoft Authentication Library (MSAL).
```

Comment:
Since there's only one sample, reword this to be more descriptive of the current contents and ideally support future additions to the table (not everyone will remember to update this intro).

If there will be only tabs apps, for example, and they're all going to be multi-platform (web, native, mobile, et al.), add that information to this intro paragraph.

Reply:
@premgan bump on this review.

Reply:
@mmacy - I should be able to complete this week, Was working on the Meta data piece first.